### ISSUE TITLE
How to set slot value automatically with predefined button payloads w/o training?

### ISSUE THREAD

**Issue by ewagner70** [RasaHQ/rasa_core#445]
**Rasa Core version**: 0.9.0a3

**Python version**: 3.6

**Operating system** (windows, osx, ...): ubuntu 18.04

**Issue**: I would like to know if there's another possibility to set slot values automatically with predefined (ideally categorical) button payloads w/o intent training or user defined Actions?

**Content of domain file** (if used & relevant):
```yaml

```

**Comment by akelad** [comment]
so you can set the payload to be something like `/inform{"entity": "value"}`

**Comment by ewagner70** [comment]
yes, I can do that.
my question was more related to NOT manually set the valuas via "/inform", but to get this out of the feedback of a user, i.e. if he simply enters only "value" as reply on a buttons template in the domain.yml, f.ex.
utter_ask_coi_gi_give_or_take:
  - text: "Do you give or receive the gift/invitation to {coi_gi_receivername}?"
    buttons:
    - title: "give"
      payload: "give"
    - title: "receive"
      payload: "receive"
I simply want to enter "give" or "receive". I've even entered a the following "training" examples in the nlu.yml:
## intent:coi_gi_give_or_take
- [give](coi_gi_give_or_take)
- [giving](coi_gi_give_or_take)
- [receive](coi_gi_give_or_take)
- [receiving](coi_gi_give_or_take)
but most of the time I get the fallback procedure ...

**Comment by akelad** [comment]
You need to label these as entities as well, doesn't look like you've done that?
also I'd suggest adding more training examples

**Comment by tmbo** [comment]
@ewagner70 yes we know, there is a lot of work to be done on the documentation. we'd love a helping hand there :wink:

**Comment by ewagner70** [comment]
@tmbo: 
would love to provide a helping hand, but for that I would need more expertise/insights ... I'm currently more experimenting via trial&error. I can offer you proof reading and "DAU validation", i.e. if I understand it, every idiot can understand it ;-)

btw - is there any kind of forum with a list of experts (for specific rasa topics) - I think that would be even more helpful than this (mostly unstructured) github issue_list.
